settings-plugin/src/main/kotlin/com/android/declarative/internal/SettingsDeclarativePlugin.kt

```diff
@@ -106,7 +113,7 @@ class SettingsDeclarativePlugin @Inject constructor(
 
         settings.pluginManagement { pluginManagement ->
             // declare our plugin version to the plugins management
-            pluginManagement.plugins.id("com.android.experiments.declarative").version(Constants.PLUGIN_VERSION)
+//            pluginManagement.plugins.id("com.android.experiments.declarative.project").version(Constants.PLUGIN_VERSION)
```

Comment:
do we still need this line?

Reply:
we don't, it's added in the root project now. removed.